File: CursoAspMVC4/Mvc4App2/Mvc4App2/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Mvc4App2.Models
{
    public class Persona
    {
        public int codigo { get; set; }
        public string nombre { get; set; }
        //apellido paterno
        public string apepat { get; set; }
        //apellido materno
        public string apemat { get; set; }


        public Persona()
        {
            // TODO: Complete member initialization
        }

        public Persona(int p1, string p2, string p3, string p4)
        {
            // TODO: Complete member initialization
            this.codigo = p1;
            this.nombre = p2;
            this.apepat = p3;
            this.apemat = p4;
        }
    }
}

File: proyectos_basic/terminal/FuncionesLambda/FuncionesLambda/Persona.vb
Public Class Persona
    Private _dni As String
    Private _nombre As String
    Private _apellido As String

    Public Sub New(_dni As String, _nombre As String, _apellido As String)
        Me._dni = _dni
        Me._nombre = _nombre
        Me._apellido = _apellido
    End Sub

    Public Property Dni As String
        Get
            Return _dni
        End Get
        Set(value As String)
            _dni = value
        End Set
    End Property

    Public Property Nombre As String
        Get
            Return _nombre
        End Get
        Set(value As String)
            _nombre = value
        End Set
    End Property

    Public Property Apellido As String
        Get
            Return _apellido
        End Get
        Set(value As String)
            _apellido = value
        End Set
    End Property
End Class
